src/Service/ReaderInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Service;
+
+/**
+ * Interface ReaderInterface
+ * @package App\Service
+ *
+ * @author Roberto Umbertini <daeronmalnwe@gmail.com>
+ */
+interface ReaderInterface
```

Comment:
This interface is not the same `Stats/Reader/ReaderInterface.php` by @garak?

Reply:
Probably @daeronmalnwe did it at the same time... Rob can you take a look?